media-video/shotcut/shotcut-20.10.31.ebuild

```diff
@@ -3,59 +3,47 @@
 
 EAPI=7
 
-inherit flag-o-matic qmake-utils xdg
+inherit qmake-utils xdg
 
 DESCRIPTION="A free, open source, cross-platform video editor"
-HOMEPAGE="https://www.shotcut.org/ https://github.com/mltframework/shotcut/"
+HOMEPAGE="https://www.shotcut.org https://github.com/mltframework/shotcut"
 SRC_URI="https://github.com/mltframework/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-BDEPEND="
-	dev-qt/linguist-tools:5
-"
-COMMON_DEPEND="
-	dev-qt/qtcore:5
+BDEPEND="dev-qt/linguist-tools:5"
+
+COMMON_DEPEND="dev-qt/qtcore:5
 	dev-qt/qtdeclarative:5[widgets]
```

Comment:
Please keep the existing ebuild style and avoid unnecessary changes, same below.